docs/index.rst

```diff
@@ -0,0 +1,20 @@
+.. Indexing_Files documentation master file, created by
```

Comment:
We already have this in [the repo](https://github.com/DesignSafe-CI/portal/blob/master/.docs/source/index.rst)